debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresType.java

```diff
@@ -88,18 +108,8 @@ public boolean equals(Object obj) {
         if (getClass() != obj.getClass())
             return false;
         PostgresType other = (PostgresType) obj;
-        if (elementType == null) {
```

Comment:
Are we totally sure that the name is unique? I guess it is, although it seems that the OID would be the safest thing to base equality on?

Reply:
Yes